specs/src/specs/data_structures.md

```diff
@@ -370,46 +369,21 @@ The blob share commitment rules may introduce empty shares that do not belong to
-| `wrappedIntermediateStateRoots` | [WrappedIntermediateStateRoot](#wrappedintermediatestateroot)`[]` | List of wrapped intermediate state roots. |
+### IndexWrapper
 
-#### WrappedIntermediateStateRoot
+IndexWrapper are wrappers around PayForBlob transactions. They include additional metadata by the block proposer that is committed to in the [available data matrix](#arranging-available-data-into-shares).
 
-| name                    | type                                            | description                                                                                                                                                                                                                                                                                                                       |
-|-------------------------|-------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `index`                 | `uint64`                                        | Index of this intermediate state root in the list of intermediate state roots. This information is lost when splitting intermediate state roots into fixed-sized [shares](./shares.md), and needs to be re-added here for fraud proof support. Allows linking an intermediate state root to a [transaction](#wrappedtransaction). |
-| `intermediateStateRoot` | [IntermediateStateRoot](#intermediatestateroot) | Intermediate state root. Used for fraud proofs.                                                                                                                                                                                                                                                                                   |
+| name            | type       | description                                                                                                                                            |
+|-----------------|------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `tx`            | `bytes`    | Actual transaction.                                                                                                                                    |
+| `share_indexes` | `[]uint32` | Share indexes (in row-major order) of the first share for each blob this transaction pays for. Needed for light verification of proper blob inclusion. |
+| `type_id`       | `string`   | Type ID of the IndexWrapper transaction type. Always set to `"INDX"`                                                                                   |
```

Comment:
I would maybe specify that this is used for encoding purposes to help ensure uniqueness

Reply:
7e11c5ce5af4f9d58bdfd13251f4cfc9d462648d